CHANGELOG.md

```diff
@@ -1,109 +1,99 @@
-## 0.10.x
+# 0.10.x
```

Comment:
however, mind making these changes to the changelog in a separate pr and explain some of the changes you're making?

Would be better for feedback since 0.10.1 is now released and there are merge conflicts